src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -704,7 +709,8 @@ private void InitializeNodeLibrary(IPreferences preferences)
 
             // Import Zero Touch libs
             var functionGroups = LibraryServices.GetAllFunctionGroups();
-            AddZeroTouchNodesToSearch(functionGroups);
+            if (!DynamoModel.IsTestMode)
+                AddZeroTouchNodesToSearch(functionGroups);
```

Comment:
As long as no test is currently relying on this (and that this change does not break all unit tests), I'm fine with this. @aosyatnik  is working on improving usage around `XPathEvaluate` on [her branch](https://github.com/DynamoDS/Dynamo/pull/4012), which I am in the process of reviewing. Then perhaps this can be added back later.